pkgs/development/tools/ronn/gemset.nix

```diff
@@ -22,6 +22,11 @@
       type = "gem";
     };
     version = "2.2.0.1";
+    # Disable randomized mangling to make ronn's outputs reproducible
+    dontBuild = false;
+    postPatch = ''
+      substituteInPlace ext/generate.c --replace '#if DEBIAN_GLITCH' '#if 1'
+    '';
```

Comment:
I'm not sure this is the appropriate way to do it. I don't have any prior experience packaging Ruby gems. However, `buildFlags = [ "-DDEBIAN_GLITCH=1" ]` didn't do the job. Apparently, the flags are not properly propagated to the C code.

Reply:
This is best handled in pkgs/development/ruby-modules/gem-config/default.nix so all packages can benefit from the fix.

In terms of changing the build flag, usually, this is done by passing the appropriate "buildFlags" attribute.